### PR TITLE
Enable package install_options

### PR DESCRIPTION
**Pull request by jfroche**
We need be able to define which repo to enable to install a package.
We expose this option with a new variable for all package resources

### CODE REVIEW

File: manifests/plugin.pp [hunk 1]
```diff
@@ -14,7 +15,8 @@
     # install from a package named "mcollective-${name}-${type}"
     $package_name = "mcollective-${name}-${type}"
     package { $package_name:
-      ensure => $package_ensure,
+      ensure          => $package_ensure,
+      install_options => $package_install_options
```
Comment (igalic) [inline manifests/plugin.pp]: puppet lint complains that you're missing a trailing comma here.

File: manifests/plugin.pp [hunk 2]
```diff
@@ -25,7 +27,8 @@
     # install the client package if we're installing on a $mcollective::client
     if $client and $has_client {
       package { "mcollective-${name}-client":
-        ensure => $package_ensure,
+        ensure          => $package_ensure,
+        install_options => $package_install_options
```
Comment (igalic) [inline manifests/plugin.pp]: puppet lint complains that you're missing a trailing comma here.

**Comment by igalic** [comment]
@jfroche this new option needs documentation, and tests.

**Comment by alexjfisher** [comment]
I think enabling repos is the sort of thing people generally expect to do in their profiles.

**Comment by jyaworski** [comment]
Hello:

What's the status of this PR? Is it still needed, and can it be squashed?

**Comment by jyaworski** [comment]
Closing due to lack of response. Please re-open if needed.

**Pull request closed without merging**